Resources/Audio/Ambience/Antag/attributions.yml

```diff
@@ -9,4 +9,8 @@
 - files: ["zombie_start.ogg"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "Made by @MIXnikita#1474 (Discord) for SS14"
-  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
\ No newline at end of file
+  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
+- files: ["headrev_start.ogg"]
+  license: "CC0-1.0"
+  copyright: "Made by https://www.youtube.com/@a-guy173"
+  source: https://www.youtube.com/watch?v=S4lhWeI5d5o
```

Comment:
This license is not valid:

1. No express license is given (that I can see) so you can't infer one you want.
2. It doesn't actually link to consent anywhere.

Reply:
the artist has a github page and in that was that lisence

Reply:
yes the github repo has a cc0 license i confirm

Reply:
https://github.com/A-Guy173/Music/blob/main/LICENSE

Reply:
i also got explicit permission from the artist

Reply:
You probably wanna link to the github file then. Not the youtube link

Reply:
linked